.github/workflows/test.yml

```
@@ -0,0 +1,38 @@
+name: Test
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
```

```suggestion
  test:
```

Or did you mean this?

---

see https://github.com/standard/standard/pull/1629#discussion_r566862061